### PR TITLE
Fixed outquotes into a model and added seed data

### PR DESCRIPTION
**Pull request by jerry1ye10**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :outquotes
```
Comment (NicholasLYang) [inline config/routes.rb]: Also make a nested resource for articles and outquotes

File: db/migrate/20170915032724_add_outquotes_to_article.rb [hunk 2]
```diff
@@ -1,5 +0,0 @@
-class AddOutquotesToArticle < ActiveRecord::Migration[5.1]
```
Comment (NicholasLYang) [inline db/migrate/20170915032724_add_outquotes_to_article.rb]: Careful. Don't get into the habit of deleting migrations. Instead, create a new migration that undoes the other's changes. If you delete migrations, then you have an issue when you try to migrate existing databases to the new schema

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20171107211226) do
+ActiveRecord::Schema.define(version: 20171107214057) do
```
Comment (jkao1) [inline db/schema.rb]: This schema has the wrong migration version right? Your latest migration is `db/migrate/20171109210835_remove_outquotes_from_articles.rb`.

**Comment by jkao1** [comment]
Fixed some drop database issues, seeds, and a syntax error in the outquote model. Merged.

**Pull request merged**